docs/faq.rst

```diff
@@ -334,6 +334,145 @@ After successfull login the domain part will be striped and the rest used as use
+
+   * Dovecot extracts the X-Spam-Level email header from the email message and converts the spam score (0 - 15) to a 0 - 100 percent scale. This spam score is compared with the user defined spam filter tolerance. If the spam score is lower than the user defined spam filter tolerance, then the email message is accepted. In logic:
+     
+     If <spam_score in percent> is greater than <spam filter tolerance>, then move the email message to the spam folder and mark the email message as read.
+   
+   For example if the user defined spam filter tolerance is set to 80%(default) and the spam score of an email message is 10:
+      
+   66% (10/15) is less than 80%, so the email is classified as ham. This email message will go to the inbox folder. If the user wants email messages with a score of 10 (66%) to be classified as spam, then the user defined spam filter tolerance can be lowered to 65% in the administration web interface.
+   
+.. image:: assets/screenshots/SpamFiltering.png
+
+The location in the administration web interface where the spam filter and spam filter tolerance can be configured.
+
+*Issue reference:* `1167`_.
+
+Can I learn ham/spam messages from an already existing mailbox?
+```````````````````````````````````````````````````````````````
+
+Mailu is supporting automatic spam learning for messages moved to the Junk mailbox. Any email moved from the Junk Folder will be learnt as ham. 
+
+If you already have an existing mailbox and want Mailu to learn them all as ham messages, you might run rspamc from within the dovecot container:
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 13 fuzzy_add /mail/user\@example.com/.Ham_Learn/cur/
+
+This should learn every file located in the ``Ham_Learn`` folder from user@example.com 
+
+Likewise, to lean all messages within the folder ``Spam_Learn`` as spam messages :
+
+.. code-block:: bash
+
+  rspamc -h antispam:11334 -P mailu -f 11 fuzzy_add /mail/user\@example.com/.Spam_Learn/cur/
+
+*Issue reference:* `1438`_.
+
+How can I block emails from a domain?
+`````````````````````````````````````
+
+Via the multimap filter it is possible to block emails from a sender domain. See the `official rspamd documentation`_ for more information. A local blacklist that contains the domains to be blocked can be configured via the Rspamd overrides folder. 
+
+The following steps have to be taken to configure an additional symbol (rule) that uses the multimap filter to block emails from sender domain.
+
+1. In the overrides folder create a configuration file for the multimap filter. This configuration is included by Rspamd in the main multimap configuration file. This means you do not have to use the "multimap {}" element. Files in the /mailu/overrides/rspamd/ folder folder are mapped to /etc/rspamd/override.d.
```

Comment:
"folder folder"

Reply:
Also, please link to the mailu overrides documentation to make more clear what this is about.

Reply:
I have re-read all text 2 times before committing this. This shows it really helps to have someone else proofread your text. Iĺl add the link to the overrides documentation.